scripts/bc-wp-init.sh

```diff
@@ -41,19 +41,30 @@ nextip(){
     echo "$NEXT_IP"
 }
 
-# 
-if [[ -n `cat /etc/hosts | grep $domain` ]]
+# WIndows and Linux have diff paths to hosts file
+case "$OSTYPE" in 
+    "linux-gnu"|"darwin")
```

Comment:
Great, but please make more generic, eg. `*linux*`, `*darwin*`, `*msys*`. This would for example not work in my case:
`➜  ~ echo $OSTYPE
darwin18.0`

Reply:
`*win*` might also make sense for windows

Reply:
Fixed in commit 9d68d7a